src/status_im/contexts/onboarding/welcome/view.cljs

```diff
@@ -45,7 +45,7 @@
           :colors [colors/neutral-100-opa-0 colors/neutral-100-opa-80]}])
       [quo/button
        {:on-press            (fn []
-                               (rf/dispatch [:init-root :shell-stack])
+                               (rf/dispatch [:update-theme-and-init-root :shell-stack])
```

Comment:
It's not the first, nor will be the last time that I think we should have some convention for names of events that are sort of private and should not be used directly. Clearly from now on, dispatching `:init-root` will lead to potential bugs, so something about the name of `:init-root` should indicate that. A comment or docstring is not as potent.

And idea for us to think about :)

Reply:
Thank you @ilmotta for reminding. I wrote the docstring, but forget to add before pushing. (Added now)